model/src/main/java/com/plhal/ares/model/DataRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.plhal.ares.model;
+
+import com.plhal.ares.model.Firma;
+
+public interface DataRepository {
+	
+	public Firma najdiFirmu(String ICO);
```

Comment:
Vim, ze jde o zkratku, ale v jave se promenne pojmenovavaji malymi pismeny. Existuje na to predpis.